CONTRIBUTING.md

```diff
@@ -74,23 +90,42 @@ yarn android
 yarn ios
 ```
 
+## Releasing
+
+To bump SDK version, run the following command:
+
+```sh
+yarn run lerna version <version> --ignore-changes --skip-git
+```
+
+This will bump versions in the `lerna.json` and related `package.json` files, they need to be committed after that.
+
+Normally [lerna version](https://github.com/lerna/lerna/tree/main/commands/version#readme) can bump version only for modules which were changed, generate changelog and push changes, but we are not using these capabilities yet.
+
+To publish packages, run the following command:
+
+```sh
+yarn run lerna publish from-package
+```
+
+This will do the publishing and also put `gitHead` to the corresponding `package.json` files.
```

Comment:
```suggestion
This publishes the package and also point the `gitHead` to the corresponding `package.json` files.
```

Reply:
It seems that replacement `point the gitHead` here doesn't work, because what is meant here is that `gitHead` field in the `package.json` will be added (or updated). So `package.json` will look like that:

```
{
    "gitHead": abcdef
}
```